geopandas/io/tests/test_pickle.py

```diff
@@ -68,43 +67,3 @@ def test_round_trip_current(tmpdir, current_pickle_data):
         result = pd.read_pickle(path)
         assert_geodataframe_equal(result, value)
         assert isinstance(result.has_sindex, bool)
-
-
-def _create_gdf():
-    return geopandas.GeoDataFrame(
-        {"a": [0.1, 0.2, 0.3], "geometry": [Point(1, 1), Point(2, 2), Point(3, 3)]},
-        crs="EPSG:4326",
-    )
-
-
-@pytest.mark.skipif(not compat.HAS_PYGEOS, reason="requires pygeos to test #1745")
-def test_pygeos_switch(tmpdir):
-    # writing and reading with pygeos disabled
-    with with_use_pygeos(False):
```

Comment:
I think Matt already mentioned it as well, but with pygeos no longer available to generate them at runtime, we should create a few of those files and commit them as test data to the repo, to ensure this keeps working.

(but that's a perfect topic for a separate follow-up PR I think)